azurelinuxagent/ga/env.py

```diff
@@ -116,16 +116,19 @@ def _operation(self):
             try:
                 if self._firewall_manager.check():
                     return  # The firewall is configured correctly
-                self._report(event.warn, "The firewall has not been setup. Will set it up.")
+                self._report(event.warn, WALAEventOperation.Firewall, "The firewall has not been setup. Will set it up.")
+            except IptablesInconsistencyError as e:
```

Comment:
Once we are in this condition, we report log every 30 seconds as env firewall setup checks every 30 seconds. Telemetry flood with this log until conntrack module issue resolved. Should we change logging period?